### ISSUE TITLE
Partner with Login.gov on a research study exploring verification at the VA

### ISSUE THREAD

**Issue by tygindraux** [department-of-veterans-affairs/va.gov-team#56881]
## Initiative

I've just been sent this email. I'm adding for visibility and will update this task once I know more about the ask or idea.

> Good morning all,
> 
> I'm reaching back out to schedule another meeting. We'd love to further discuss a research study exploring identity verification at VA. We'd love to talk about our goals for the research and what time frame might work best. I'm including Ponja, who has recently moved from Team Katherine (authentication) to Team Ada to work on UX research for identity verification. 
> 
> Would you be free for a 60-minute meeting sometime next week? Below are a few free times for a meeting:
> Monday, April 24, 3 - 5 pm ET
> Tuesday, April 25, 10 - 11 am or 2:00 - 3:30 pm ET
> Wednesday, April 26, 10 - 11 am or 2 - 5 pm ET
>  Thursday, April 27, 10 - 11 am or 2 - 5 pm ET
> Please let us know which of these times works best. Thank you!
> 
> Princess

## Tasks
- [x] attend meeting with Login.gov on Apr 19
- [x] next steps after meeting with Login.gov on Apr 24

**Comment by alevindecanini** [comment]
@tygindraux can you point us toward any notes / next steps from the Login.gov research planning meeting yesterday?

**Comment by tygindraux** [comment]
@alevindecanini – Sorry, I posted in Slack. But next steps was another meeting to continue the conversation (which happened today). Basically they're interested in learning more about how a user is prepared for identity verification before they get sent to Login.gov. I shared our latest research as well as Clayton's research plans. (And told them that this is something we're looking at on our end.) I also pressed to learn more about what happens on their end of the identity verification flow eg. we know when people drop off, but I'd like to understand more about why they drop off. That would potentially help us prepare people properly (if that's the issue) or solve for the right problems. We all think collaborating on a study that looks at the full flow (including our product and theirs) would be worthwhile, but we first want to take a step back to understand what knowledge is missing. Part of that will come from Clayton's study. I'm going to close out this ticket, but there's room for going back to Princess' team at Login.gov when we're ready for more research into the identity verification (online) flow.
